src/pages/creer-un-jeu/mimique/index.tsx

```diff
@@ -8,16 +8,24 @@ import { Base } from 'src/components/Base';
 import { useGameRequests } from 'src/services/useGames';
 import { GameType } from 'types/game.type';
 
-const Mimique: React.FC = () => {
+const Mimique = () => {
   const router = useRouter();
-  const { getAvailableGamesCount } = useGameRequests();
+  const { getIsGameCreateBeforePlaying, getAvailableGamesCount } = useGameRequests();
   const [mimicsCount, setMimicsCount] = React.useState<number>(0);
+  const [isGameCreateBeforePlaying, setIsGameCreateBeforePlaying] = React.useState<boolean>(false);
 
   React.useEffect(() => {
-    getAvailableGamesCount(GameType.MIMIC).then((count) => {
-      setMimicsCount(count);
-    });
-  }, [getAvailableGamesCount]);
+    Promise.all([
+      getIsGameCreateBeforePlaying(GameType.MIMIC).then((game) => {
+        if (game > 0) {
+          setIsGameCreateBeforePlaying(true);
+        }
+      }),
+      getAvailableGamesCount(GameType.MIMIC).then((count) => {
+        setMimicsCount(count);
+      }),
+    ]);
+  }, [getAvailableGamesCount, getIsGameCreateBeforePlaying, isGameCreateBeforePlaying]);
```

Comment:
- The `Promise.all` is not necessary here because you are not using it.
- I would use 2 separate useEffects because they are not the same API requests.
- `isGameCreateBeforePlaying ` This name is very difficult to understant. I would name it: `hasUserCreatedMimics`.
- `getIsGameCreateBeforePlaying` This name is very difficult to understand. I would name it: `getUserCreatedGamesCount`.

**Suggestion:**
```tsx
React.useEffect(() => {
    getAvailableGamesCount(GameType.MIMIC).then((count) => {
      setMimicsCount(count);
    });
  }, [getAvailableGamesCount]);

React.useEffect(() => {
    getUserCreatedGamesCount(GameType.MIMIC).then((game) => {
      setHasUserCreatedMimics(game > 0);
    });
}, [getUserCreatedGamesCount]);
```